src/lib/tecs/TECS.hpp

```diff
@@ -272,6 +272,7 @@ class TECSControl
 		float altitude_rate;	///< Current altitude rate of the UAS [m/s].
 		float tas;		///< Current true airspeed of the UAS [m/s].
 		float tas_rate;		///< Current true airspeed rate of the UAS [m/s²].
+		float pitch; 		///< Current vehicle pitch angle [rad].
```

Comment:
Is this the absolute pitch or the pitch above FW_PSP_TRIM?

Reply:
Should be the pitch above FW_PSP_TRIM:
https://github.com/PX4/PX4-Autopilot/blob/main/src/modules/fw_pos_control/FixedwingPositionControl.cpp#L2777

Reply:
I think it's above FW_PSP_TRIM --> could be specified in the comment, but actually not just here but for all "pitch" states in here.